howto/stream/exchange-oob-data.md

```diff
@@ -269,6 +272,55 @@ try {
     ex.printStackTrace();
 }
 ```
+
+#### Install the APK as system app
+
+To connect the data channel to the Anbox WebRTC data proxy service within an Android container, the Android app must be installed and running as a system app. To do so, proceed with the following steps:
+1. Add the attribute `android:sharedUserId="android.uid.system"` into the `<manifest>` tag in the `AndroidManifest.xml` file of your Android app
+2. Create an addon to install your APK as a system app through the pre-start hook with the following content:
```

Comment:
Why can't we use a raw instance instead? Adding the need for addons IMHO just makes things complicated here.

Reply:
That's a good idea.

Reply:
Hmm, no, we can't. Using the raw instance makes it impossible to run a pre-start hook where the APK is going to be installed as a system app. I suppose what you meant above was to use the application hooks, but that makes no difference to me. The example below already uses the raw instance.